ext/nnstreamer/tensor_filter/tensor_filter_movidius_ncsdk2.c

```diff
@@ -281,13 +281,56 @@ _mvncsdk2_open (const GstTensorFilterProperties * prop, void **private_data)
  * @param[in] input : The array of input tensors
  * @param[out] output : The array of output tensors
  * @return 0 if OK. non-zero if error.
- * @todo : fill this function
  */
 static int
 _mvncsdk2_invoke (const GstTensorFilterProperties * prop, void **private_data,
     const GstTensorMemory * input, GstTensorMemory * output)
 {
+  ncStatus_t ret_code;
+  mvncsdk2_data *pdata = *private_data;
+  guint32 buf_size;
+
+  /* Warning: conversion unsigned long to unsigned int */
+  buf_size = (guint32) input->size;
+  ret_code = ncFifoWriteElem(pdata->handle_fifo_input, input->data,
```

Comment:
params const GstTensorMemory * input and GstTensorMemory * output

Currently input[16] output[16] passed from tensor_filter main.
This is ok here if this handles only one input/output memory block.

Reply:
If you want to support single-tensor only, you need to generate warning message or error if multi-tensor is given.

Reply:
@myungjoo IMHO, since num_tensors have been set to NNS_MVNCSDK2_MAX_NUM_TENOSORS_SUPPORTED (i.e., 1) at the stages of getInputDimension and getOutputDimension, the input of this function would be always single-tensor. I will add a g_assert here for just in case.